### PR TITLE
OCRAD-18 :- add concept data type document.

### PR DESCRIPTION
**Pull request by Akayeshmantha**

### CODE REVIEW

File: Concepts/concept_data_type.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    If user not logged in to perform this action,a `401 Unauthorized` status returned.
+    
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Display Name of concept data Type.
+
+    ```console
+    GET /conceptdatatype?q="Search Query"
```
Comment (bmamlin) [inline Concepts/concept_data_type.md]: Does this work for you? In my testing, I get an empty list back if I try to search (i.e., any value for `q`).

File: Concepts/concept_data_type.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+     ```
+    
+* #### List concept data type by UUID.
```
Comment (bmamlin) [inline Concepts/concept_data_type.md]: "Get" (not "List")

**Review by bmamlin — CHANGES_REQUESTED** [review]

File: Concepts/concept_data_type.md (hunk 1 above)
Comment (Akayeshmantha) [reply Concepts/concept_data_type.md]: Hi @bmamlin executing without any q filter it works and gives a list. But the q filter is not working.

Comment (bmamlin) [reply Concepts/concept_data_type.md]: Okay. So, let's not document a feature that isn't supported. Document the ability to get all concept data types, but don't include a search parameter in this case.

Comment (Akayeshmantha) [reply Concepts/concept_data_type.md]: sure will do

**Comment by Akayeshmantha** [comment]
@bmamlin  done the changes.

**Pull request merged**